src/main/flight/mixer.c

```diff
@@ -832,6 +848,10 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensa
     throttle = pidCompensateThrustLinearization(throttle);
 #endif
 
+#ifdef USE_DYN_IDLE
+    throttle += currentPidProfile->idle_throttle * 0.001f;
```

Comment:
Should precompute `currentPidProfile->idle_throttle * 0.001f;` and keep in `FAST_RAM`.

Reply:
I modeled this after other uses of the currentPidProfile in the same file. Such as 
```
    const float scaledAxisPidRoll =
        constrainf(pidData[FD_ROLL].Sum, -currentPidProfile->pidSumLimit, currentPidProfile->pidSumLimit) / PID_MIXER_SCALING;
```

I thought not precomputing values was ok here since this code only runs once per pid loop (unlike code that runs per axis or per motor). I'm not sure how tight FASTRAM is - if we have plenty it might be a good idea to precompute all values even if used only once per pidloop.

Reply:
likely that should be optimized/precomputed too.  anything that runs ~8000 times a second is probably a good candidate for pre-computation, but lower down the optimization pile than motor code/axis which as you mentions runs even more frequently.

Reply:
Ok - moved this stuff to init with shadow fastram floats.